### DISCUSSION TITLE
Interesting tool to convert time series into features

### DISCUSSION THREAD

**Discussion by PeterPirog** [feature-engine/feature_engine#487] [Ideas]
Lst time I found interesting tool to convert time series into features.
Maybe it will useuseful in the project.

[https://github.com/facebookresearch/Kats/blob/main/tutorials/kats_203_tsfeatures.ipynb](https://github.com/facebookresearch/Kats/blob/main/tutorials/kats_203_tsfeatures.ipynb)

**Comment by solegalli** [comment]
FYI @KishManani
